### ISSUE TITLE
State normalization is implemented twice, and the copies disagree

### ISSUE THREAD

**Issue by TheWisp** [TheWisp/lerobot#235]
Twenty policies normalize observation state through `NormalizeProcessorStep`, taking statistics from `dataset.meta.stats`. HVLA does not. It has no `normalization_mapping`, and computes its own mean and std over the frames its trainer happened to load.

The two disagree most where it matters least to notice: a joint the task leaves still.

## What it costs

On `thewisp/0803_20260803_174402` (241 episodes, 40,101 frames) the whole left arm is held still. Every left `.pos` channel has a dataset-wide std under 0.13:

```
left_joint_2.pos  0.00094     left_joint_5.pos  0.1244
left_joint_3.pos  0.0016      left_joint_6.pos  0.0946
left_joint_1.pos  0.0362      left_joint_7.pos  0.0099
left_joint_4.pos  0.0877      left_gripper.pos  0.0195
```

A policy normalizing from `dataset.meta.stats` divides `left_joint_3.pos` by 0.0016. HVLA, computing over its own frame subset where that joint never moved at all, gets exactly zero and clamps to its `1e-6` guard — 1600× smaller. A rig reading 0.22° from the training mean then normalizes to about 137 sigma under the shared statistics and about 218,000 under HVLA's. One channel that size dominates the first linear layer and corrupts the output for every joint, including the ones doing the work.

This is why the same data trained fine under SmolVLA and did not under HVLA. It is a difference in where the statistics come from, not in the data.

## It is already solved, once

`NormalizeProcessorStep`'s `QUANTILES` path floors the denominator and clamps the result, and its comment names this dataset's own number:

```python
# Use 0.1 threshold to catch small denominators like 0.001604 that cause explosions
denom = torch.where(torch.abs(denom) < 0.1, min_denom, denom)
...
return torch.clamp(normalized, -10.0, 10.0)
```

`0.001604` is `left_joint_3.pos`. The dataset already carries `q01/q10/q50/q90/q99`.

#233 shipped a second copy of that mechanism inside HVLA — a floor on `*.pos` std plus a ±10 clamp, the same bound the processor uses, arrived at independently. It works, and it is narrower: being expressed in dataset units it reaches position channels only, so velocity and torque channels stay unbounded and rely on the clamp alone. The quantile path has no such restriction.

## The fork worth recording

Moving HVLA onto the processor removes the duplicate and the restriction. It also reaches 15 files and 78 references, including the checkpoint format, `s1_inference`, `rlt/intervention`, the flash-dagger scripts and `saliency`. Every existing HVLA checkpoint stores mean/std; the quantile path wants q01/q99, so code and checkpoints stop matching. `scripts/hvla_migrate_checkpoints.py` exists, so there is precedent for moving them.

Leaving it as it is keeps two implementations of one rule, which is how one gets fixed and the other does not.

The decision this needs is whether HVLA's trainer has a reason to compute statistics over its own frame subset rather than read the dataset's. Nothing in its history records one — the behaviour arrived whole with `ed4b1fda1 feat(hvla): swappable S1 with flow matching`, alongside its own dataset wrapper and training loop. If there is no reason, the duplicate is an accident of that implementation being written standalone.

Related: #233 (the HVLA-local floor and clamp), `src/lerobot/policies/hvla/s1/flow_matching/normalization.py`, `src/lerobot/processor/normalize_processor.py`.
